src/gfx/shaders/Uber_vert.glsl

```diff
@@ -218,12 +219,19 @@ void main() {
 
 #if defined(USE_LIGHTS) && defined(SHADOWMAP)
 	#if NUM_DIR_LIGHTS > 0
-	vec4 worldPosition;
-	#pragma unroll_loop
-	for ( int i = 0; i < NUM_DIR_LIGHTS; i ++ ) {
-	  worldPosition = vec4(vWorldPosition, 1.0);
-		vDirectionalShadowCoord[ i ] = directionalShadowMatrix[ i ] * worldPosition;
-	}
+	  vec4 worldPosition;
+	  #pragma unroll_loop
```

Comment:
The  same problem with unroll_loop here.

Reply:
The same answer